### PR TITLE
Adding EventSub Channel Points Mock Events

### PR DESCRIPTION
**Pull request by TheHellcat**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

## Problem/Feature

This PR adds the funcionality of creating mock events for Channel Points rewards and redemptions.

## Description of Changes: 

- Added `twitch-cli event trigger add-reward`
- Added `twitch-cli event trigger update-reward`
- Added `twitch-cli event trigger remove-reward`
- Added `twitch-cli event trigger add-redemption`
- Added `twitch-cli event trigger update-redemption`

## Checklist

- [x] My code follows the [Contribution Guide](https://github.com/twitchdev/twitch-cli/blob/master/CONTRIBUTING.md#Requirements)
- [x] I have self-reviewed the changes being requested
- [x] I have made comments on pieces of code that may be difficult to understand for other editors
- [x] I have updated the documentation (if applicable)

### CODE REVIEW

File: cmd/events.go [hunk 1]
```diff
@@ -63,6 +66,9 @@ func init() {
 	triggerCmd.Flags().BoolVarP(&isAnonymous, "anonymous", "a", false, "Denotes if the event is anonymous. Only applies to Gift and Sub events.")
 	triggerCmd.Flags().IntVarP(&count, "count", "c", 1, "Count of events to trigger. This will simulate a sub gift, or large number of cheers.")
 	triggerCmd.Flags().StringVarP(&secret, "secret", "s", "", "Webhook secret. If defined, signs all forwarded events with the SHA256 HMAC.")
+	triggerCmd.Flags().StringVarP(&status, "status", "S", "", "Status of the event object, currently applies to channel points redemptions.")
+	triggerCmd.Flags().StringVarP(&objectId, "object-id", "i", "", "Manually set the ID of the event payload object (for example the reward ID in redemption events).")
```
Comment (lleadbet) [inline cmd/events.go:70]: Do you mean the ID of the channel points object?

Comment (lleadbet) [reply cmd/events.go:70]: If so- I believe that item-id might be a bit better for outside users- object-id is a bit obtuse in my opinion.

**Review by lleadbet — COMMENTED** [review]
One minor question noted, but otherwise lgtm.

File: cmd/events.go (hunk 1 above)
Comment (TheHellcat) [reply cmd/events.go:70]: Yeah, I wanted to name it as generic as possible, so the parameter can be reused for other calls, in the future.
But "item-id" indeed sounds like a better name.

**Comment by TheHellcat** [comment]
Change of "object ID" to "item ID" has been pushed :-)

**Review by lleadbet — APPROVED** [review]

**Pull request merged**